### DISCUSSION TITLE
line 7: syntax error near unexpected token `newline'

### DISCUSSION THREAD

**Discussion by dannyolsen** [TinCanTech/easy-tls#276] [EasyTLS and related HELP]
hi,

i get following error when using ./easytls on a raspberry pi running ubuntu

root@ubuntu:/easy-rsa# root@ubuntu:/easy-rsa# ./easytls
./easytls: line 7: syntax error near unexpected token `newline'
./easytls: line 7: `<!DOCTYPE html>'

anyone?

**Comment by TinCanTech** [comment]
Linked: https://github.com/TinCanTech/easy-tls/issues/277

This is probably a bug I can fix.

**Comment by dannyolsen** [comment]
Hi and thank you…

 

By the way  - I work for Wago and I am trying to connect our PLC to openVPN ( which I know is possible ) but I am a bit lost in all the keys and all – would you by any chance know someone who would be willing to help me out…

 

I have tried soooooo many youtube videos and tutorials, but I there is always a breaking point for me so it doesn’t work and my knowledge is a bit limited on this matter… I have been trying for three days now..

 

 

I have the Openvpn access server up and running on a  DNS on a raspberry pi running ubuntu 20.04..

 

Now I am trying to create server certs and all for my PLC and as a test for my windows machine…

 

 

Any help would be appreciated

 

 

Thanks you!

**Comment by TinCanTech** [comment]
This is the Easy-TLS discussion board.

If you need assistance with your Company VPN then I am available **for hire**.

**Comment by dannyolsen** [comment]
Hi Tincan 😉

 

Yes maybe hiring you would be and option, I spoke to my boss already – how much do you charge pr. Hour and pr. Day? I guess this would be like a day..

 

My needs are following:

*	Setup and openvpn server on ubuntu raspberry ( which I have tried but I am unable to connect my windows machines to it )
*	Create CA authority, Server certificate and client certificate

*	Client certificate for my windows 11 PC
*	Client certificate for our Wago PLC 750-8212 which accepts .ovpn. ( I do have a older step by step on how to do this, but I cant make it work )
*	Connect clients to server

 

I have some knowledge but limited so bear with me. 

 

Regards

 

Danny

***@***.***

**Reply by TinCanTech** [reply]
You can contact me privately on my email account tincantech at protonmail dot com
